ext/standard/tests/general_functions/bug77638_2.phpt

```diff
@@ -2,6 +2,8 @@
 Bug #77638 (var_export'ing certain class instances segfaults)
 --EXTENSIONS--
 ffi
+--XFAIL--
+var_export now expect consistent properties
```

Comment:
Introduction of segfaults in existing tests is unacceptable of course.

Reply:
Sure, this is just a draft so I would of course fix all the issues before it's ready. I mainly wanted to check if the approach looks fine to you before I finalize it.